app/decorators/content_item_decorator.rb

```diff
@@ -9,6 +9,10 @@ def last_updated
     end
   end
 
+  def feedex_link
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/107/commits/a7dbf53cb2899cb9520fe9548fcaa3b6890d06b9#diff-d996e508524ccdda36fc9ba9389ec24fR12